shenyu-web/src/main/java/org/apache/shenyu/web/loader/ShenyuPluginPathBuilder.java

```diff
@@ -30,7 +30,7 @@ public final class ShenyuPluginPathBuilder {
 
     private static final String PLUGIN_PATH = "plugin-ext";
     
-    private static final String DEFAULT_EXT_PLUGIN_PATH = "/ext-lib/";
+    private static final String DEFAULT_EXT_PLUGIN_PATH = "ext-lib/";
```

Comment:
Why can't  use '/ ' at the beginning?

Reply:
The absolute path of `/ext-lib/` is not in the deployment directory. On Linux, it's just pointing to the `/ext-lib` folder. On Windows it is in the root of the deployment driver. A log point added to confirm this. We can find this in test case too: 
https://github.com/apache/shenyu/blob/1f5b5cda230c10b577712918db95c4028d902345/shenyu-web/src/test/java/org/apache/shenyu/web/loader/ShenyuPluginPathBuilderTest.java#L42-L49